src/aleph/vm/orchestrator/supervisor.py

```diff
@@ -101,6 +102,7 @@ def setup_webapp():
         # /control APIs are used to control the VMs and access their logs
         web.post("/control/allocation/notify", notify_allocation),
         web.get("/control/machine/{ref}/logs", stream_logs),
+        web.get("/control/machine/{ref}/logs2", operate_logs),
```

Comment:
I think that will be better to maintain the same endpoint and detect if we are using Websockets or not.

Reply:
I agree with this.

Reply:
I'm not too sure the behaviour are quite different, one is streaming and returning json, the other is only sending existing logs as text. We could merge both and have some kind of behemoth that can handle everything (text/json/streaming and not streaming) but that would multiply the complexity

Reply:
it's doable but it  is getting a bit out of scope for  a simple debug endpoint

Reply:
So maybe use a different endpoint path, like `steam_logs` for WebSockets endpoint and normal `logs` for REST endpoint.